index.js

```diff
@@ -39,35 +40,23 @@ module.exports = function (content) {
     }
 
     var filePath = prefix + resource.slice(relativeToIndex + relativeTo.length); // get the base path
-    var html;
+    var html, match = content.match(/^module\.exports.+?(".*")/);
```

Comment:
yes, it does not support single quotes now.

i just realize you support it before.
but it has a potential bug, that you don't consist on the quote type

And i think that's rare case some loader or webpack return single quoted string.